style.css

```diff
@@ -0,0 +1,206 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-weight: bold;
+}
+
+hr {
+  color: #DDD;
+}
+
+.main > .feed > .feed-post > .feed-profile-pic,
+.main > .feed > .feed-post > div > div,
+.main .compose-grid > div { 
+  margin: 4px;
+  border: 3px solid black;
+  border-radius: 7px;
+}
+
+body > section {
+  height: auto;
+}
+
+body {
+  min-height: 100vh;
+  display: grid;
+  grid-template-columns: 200px 1fr 300px;
+  column-gap: 10px;
+  margin-left: 100px;
+  margin-right: 100px;
+  height: auto;
+}
+
+.navigation {
+  display: grid;
+  grid-template-rows:  60px 60px 60px 60px 60px 60px 60px 60px 60px 60px 1fr 90px;
+}
+
+.navigation > div {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: left;
+  align-items: center;
+  padding: 10px;
+  border-radius: 10px;
+}
+
+.navigation > .tweet-button {
+  padding: 0;
+}
+
+.main .compose-grid > .compose-post > .post-attachments > div > img:hover,
+.navigation > div + div:hover {
+  background-color: #EEE;
+}
+
+.main .compose-grid > .compose-post > .post-attachments > div > img:active,
+.navigation > div + div:active{
+  background-color: #CCC;
+}
+
+.navigation > div + div + div + div + div + div + div + div + div + div + div {
+  z-index: -1;
+}
+.navigation > div + div + div + div + div + div + div + div + div + div + div:hover {
+  background-color: white;
+}
+.navigation > div + div + div + div + div + div + div + div + div + div + div:active {
+  background-color: white;
+}
```

Comment:
I feel like you know this is bad! I get what you're doing (after you explained it) but even so you could have used nth:child to make it much cleaner.